starter-workflows/code-scanning/frogbot-fix.yml

```diff
@@ -0,0 +1,55 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: "Frogbot Fix"
+on:
+  push:
+    branches:
+      # Creating fix pull requests will be triggered by any push to one of the these branches.
+      # You can add or replace to any branch you want to open fix pull requests for.
```

Comment:
You can add or replace to any branch you want to open fix pull requests for.
-->
You can edit the list of branches you wish to open fix pull requests on.

Reply:
done